scripts/builtin/mice_linearReg.dml

```diff
@@ -0,0 +1,147 @@
+#-------------------------------------------------------------
+#
+# Modifications Copyright 2019 Graz University of Technology
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+# Assumptions
+# 1. The data is continous/numerical 
+# 2. The missing values are denoted by zeros
+
+# Builtin function Implements binary-class SVM with squared slack variables
+#
+# INPUT PARAMETERS:
+# ---------------------------------------------------------------------------------------------
+# NAME            TYPE    DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# X               Double   ---         matrix X of feature vectors
+# iter            Integer    3         Number of iteration for multiple imputations 
+# complete        Integer    3         A complete dataset generated though a specific iteration
+# ---------------------------------------------------------------------------------------------
+ 
+
+#Output(s)
+# ---------------------------------------------------------------------------------------------
+# NAME                  TYPE    DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# dataset               Double   ---        imputed dataset
+# singleSet             Double   ---        A complete dataset generated though a specific iteration
+
+
+
+m_mice_linearReg = function(Matrix[Double] X, Integer iter = 3, Integer complete = 3)
+return(Matrix[Double] dataset, Matrix[Double] singleSet)
+  {
+  n = nrow(X)
+  row = n*complete;
+  col = ncol(X);
+  R = matrix(0, rows=1, cols=col)
+  MR = matrix(0, rows=1, cols=col)
+  
+  mu = colMeans(X) #mean of columns for initial imputation
+
+# storing the mask/address of missing values
+  Mask = X==0;
+  Mask_Filled = Mask;
+
+# filling the missing data with thier means
+  X2 = X+(Mask*mu)
```

Comment:
You can replace mu with colMeans(X) directly and remove a few unnecessary line spaces throughout.
Please fix the typo in the comment.